utils/posts.ts

```diff
@@ -31,7 +31,7 @@ export async function getPost(slug: string): Promise<Post | null> {
     return {
       slug,
       title: attrs.title as string,
-      publishedAt: new Date(attrs.published_at as Date) || null,
```

Comment:
turns out that `|| null` wouldn't work here.  
even though `new Date(undefined)` returns `Invalid Date` it'd still be a truthy value.
<img width="217" alt="Bildschirm­foto 2023-06-26 um 21 37 34" src="https://github.com/denoland/saaskit/assets/8133892/87ab534a-045e-4b3d-ac79-187cb46a0790">